src/VerifyResourceUsage.pl

```diff
@@ -1,13 +1,13 @@
-#!/usr/bin/perl
 ######################################################################
-# verify_resource_usage.pl
+# VerifyResourceUsage.pl
 ######################################################################
-# run like this:
+# Run like this:
 #    perl VerifyResourceUsage.pl
-# to discover 
+# To discover 
 #    1) what LogMessages.cs strings are unused
 #    2) whether any of the string IDs and string value prefixes don't match
 #    3) whether any LogMessages.cs files contain duplicate IDs
+#    4) whether any commented out IDs have been accidentally reused again
```

Comment:
>again [](start = 68, length = 5)

nit: again might not be required in the sentence

Reply:
True; fixed this!